index.html

```diff
@@ -36,14 +40,25 @@
           </div>
         </script>
 
-        <!--<script type="infowindow/html" id="hover_template">
-          <div class="cartodb-tooltip-content-wrapper">
-              <div class="cartodb-tooltip-content">
-                <span>{{school}}</span>
-                <span>{{ytd_money}}</span>
-              </div>
+        <script type="infowindow/html" id="infowindow_template_student">
+          <div class="infowindow cartodb-popup v2">
+            <a href="#close" class="cartodb-popup-close-button close">x</a>
+             <div class="cartodb-popup-content-wrapper">
+               <div class="cartodb-popup-content">
+                <div class="school-name">{{content.data.schnam}}</div>
+                <div class="info-row">
+                    <span class="spending-label">total spending</span>
+                    <span class="spending">{{content.data.ytd_money}}</span>
+                </div>
+                <div class="info-row">
+                    <span class="spending-label">per student</span>
+                    <span class="spending highlight">{{content.data.per_student_money}}</span>
+                </div>
+               </div>
+             </div>
+             <div class="cartodb-popup-tip-container"></div>
```

Comment:
Having two templates just for this sucks. Should see if there's a better cartodb way